models/stg_marketo__activity_merge_leads.sql

```diff
@@ -22,9 +22,9 @@ with base as (
         cast(activity_date as {{ dbt_utils.type_timestamp() }}) as activity_timestamp,
         activity_type_id,
         campaign_id,
-        lead_id,
+        cast(lead_id as {{ dbt_utils.type_int() }}) as lead_id,
         master_updated,
-        trim(trim(merge_ids,']'),'[') as merged_lead_id,
+        cast(replace(trim(trim(merge_ids,']'),'['), ',', ', ') as {{ dbt_utils.type_string() }}) as merged_lead_id,
```

Comment:
Nice job combining unnecessary downstream steps into one within the staging model!

Reply:
Good callout thanks! I think it was a leftover of combining logics, but it makes sense they should both be ints after all the trims. I tested it out and have updated the type of line 27 to int.

Reply:
Revision to previous comment--actually it should stay as it was. For line 25 it is a single number, but for line 27 it can be a string of several IDs separated by a comma.